v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,472 @@
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+cockroach sql --url $CLI_DB_CONNECTION_URL --execute "show users;"
+~~~
+
+~~~txt
+                       username                       |                options                | member_of
+------------------------------------------------------+---------------------------------------+------------
+  admin                                               |                                       | {}
+  root                                                |                                       | {admin}
+  v-token-hc-crdb-rol-uqtqyca6neplilakhv55-1653513157 | VALID UNTIL=2022-05-25 22:12:42+00:00 | {}
+  v-token-hc-crdb-rol-0gzk3hwi7k7a9w1ggggq-1653515525 | VALID UNTIL=2022-05-25 22:52:10+00:00 | {}
+  v-token-hc-crdb-rol-varnevyjgjg9zgf62kps-1653515522 | VALID UNTIL=2022-05-25 22:52:07+00:00 | {}
+  v-token-hc-crdb-rol-wltbv25napuroomvzmok-1653515524 | VALID UNTIL=2022-05-25 22:52:09+00:00 | {}
+~~~
+
+### Revoke credentials
+
+Revoke a lease or two just for the feeling of raw power. Recall that a "lease" on a "role" in Vault terms corresponds to an actual credential pair on the CockroachDB cluster, i.e., a username/password combination for a SQL user.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault lease revoke database/creds/crdb-role/XIpIBRM8FkQxD5B0ndB1lszY.iMXtW
+~~~
+
+~~~txt
+All revocation operations queued successfully!
+~~~
+
+### Provision a CockroachDB-client Vault policy
+
+As admin, provision a Vault policy for CockroachDB client operators to access the client credentials.
+
+The purpose of the previous work is to make a dynamic secret that can be used to access the CockroachDB database by generating credentials on demand. Performing the above work (establishing the connection between the CockroachDB cluster and the Vault cluster, creating the template for database client credentials, etc.) required admin privileges. But for the work to be meaningful, a Vault user with more limited permisions must be able to access the generated credentials.
+
+This policy will be used to access CockroachDB client credentials. In keeping with the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege), let's give it only the required ability to read the required credential. 
+
+Vault policies are specified using HashiCorp Configuration Language (HCL). The following configuration specifies a policy of read access for the `crdb-role` credential.
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+vault policy write roach-client - <<hcl
+path "database/creds/crdb-role" {
+  capabilities = [ "read" ]
+}
+hcl
+~~~
+
+~~~shell
+Success! Uploaded policy: roach-client
+~~~
+
+### Generate an authentication token for the `crdb-role` Vault user
+
+Our final act as Vault admin will be to provision an authentication token to assume the `crdb-role`, which represents the Vault role with the sole purpose of providing credentials ot a CockroachDB client.
```

Comment:
There is a typo on "to" here. Also, it might be worth breaking this sentence into two to remove any complexity in the sentence. 

When you say "represents" do you mean "it is"? Or not? If it is, maybe something like the following:

"Finally, as Vault admin you will provision an authentication token to assume the `crdb-role`. This Vault role has the sole purpose of providing credentials to a CockroachDB client."